lumicks/pylake/kymo.py

```diff
@@ -91,6 +132,22 @@ def timestamps(self) -> np.ndarray:
     def _plot(self, image, **kwargs):
         import matplotlib.pyplot as plt
 
+        def fetch_custom_property(args, field):
+            """Grabs a field from the input arguments and consumes it.
+
+            Parameters
+            ----------
+            args : dict
+                Dictionary with function arguments
+            field : string
+                Field to look for
+            """
+            if field in args:
+                tmp = args[field]
+                args.pop(field)
```

Comment:
can be tmp = args.pop(field) ?

Reply:
Good point. Could even return args.pop(field) directly. Fixed 👍